File: assignment 19/p4.c
#include<stdio.h>
#include<stdlib.h>
void Digit(int Arr[],int isize)
{
int icnt=0,counter=0;
for(icnt=0;icnt<isize;icnt++)
{
if((Arr[icnt]>=100&&Arr[icnt]<=999))
{
printf("%d ",Arr[icnt]);
}
else
{
	if((Arr[icnt]<=-100&&Arr[icnt]>=-999))
{
printf("%d ",Arr[icnt]);
}
}
}
}
int main()
{
int icnt=0,ilength=0;
int *ptr=NULL;
printf("how many elements you want");
scanf("%d",&ilength);
ptr=(int*)malloc(sizeof(int)*ilength);
if(ptr==NULL)
{
printf("malloc fails ");
return -1;
}
printf("enter elements\n");
for(icnt=0;icnt<ilength;icnt++)
{
scanf("%d",&ptr[icnt]);
}
Digit(ptr,ilength);
free(ptr);
return 0;
}

File: assignment 32/P4.java
import java.util.*;
class Mangesh
{
	public void Display(int arr[],int ino1,int ino2)
	{
		int i=0,isize=0;
		isize=arr.length;
		for(i=0;i<isize;i++)
		{
			if(arr[i]%ino1==0&&arr[i]%ino2==0)
			{
				System.out.println(arr[i]);
			}
			
		}
		
	}
	
}
class P4
{
	public static void main(String[] args) 
	{
		Scanner sc = new Scanner(System.in);
		int ilen=0, icnt=0,ino1=0,ino2;
		System.out.println("enter divisor 1");
		ino1=sc.nextInt();
		System.out.println("enter divisor 2");
		ino2=sc.nextInt();
		System.out.println("how many elements you want");
		ilen=sc.nextInt();
		int brr[]=new int[ilen];
		System.out.println("enter no");
		for(icnt=0;icnt<ilen;icnt++)
		{
			brr[icnt]=sc.nextInt();
		}
		Mangesh mobj= new Mangesh();
	   System.out.println("no divisable by divisor which are even are");
		mobj.Display(brr,ino1,ino2);
		
	}
}